octodns_cloudflare/__init__.py

```
@@ -728,6 +744,14 @@ def _record_is_just_auto_ttl(self, record):
             and record._octodns.get('cloudflare', {}).get('auto-ttl', False)
         )
 
+    def _record_comment(self, record):
+        'Returns record comment'
+        return record._octodns.get('cloudflare', {}).get('comment', '')
+
+    def _record_tags(self, record):
+        'Returns record tags'
+        return sorted(record._octodns.get('cloudflare', {}).get('tags', []))
```

Does CF allow duplicate tag values, e.g. ['one', 'one']? I see that things are sorted which should make this compare consistently for the most part, but I'm guessing they don't allow duplicate values. If not it may be preferable to treat them as a set internally rather than sorting and only have them as a list in YAML (and when pushed up to CF.

---

CF silently drops the duplicate on their end. I agree with treating this as a set instead and will make the proposed changes.

---

@ross , updated and ready for review when you have a moment.